portfolio-project-matching-app/components/Textarea.js

```diff
@@ -0,0 +1,14 @@
+const Textarea = ({ form_id, name, placeholder, numRows, value, addClassName='' }) => {
```

Comment:
Still a hunk of styling needed here. I used Charlie's text input as inspiration.